.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+
+[*.json]
+indent_size = 2
+
+[*.{markdown,md}]
+indent_size = 4
+max_line_length = 80
+trim_trailing_whitespace = false
+
+[*.py]
+indent_size = 4
+max_line_legth = 120
+
+[*.{yaml,yml}]
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
did you set this to tab to match existing indent?  should we change to spaces?

Reply:
Yes, I didn't intend to change what was already there. Also I believe tabs are the standard syntax for Makefiles. Depending on which version of `make` you have changing it to spaces might actually break the Makefile for some folks so I'd recommend keeping it as tabs.

Reply:
ok